### ISSUE TITLE
Adding information on the main settings screen

### ISSUE THREAD

**Issue by Krzysiu** [jacquesh/foo_openlyrics#441]
I propose adding something on the main screen of settings, so dumbos, like me, would have it easier. Something like "Go to the subitems for settings"

It would also improve accessibility because, at least for me, that gray > which shows, there's more, is barely visible. And while this should be changed by FB2000 team, for now that would be a nice touch. It's not breaking anything, everything will be just the same, just a bit easier.

![Image](https://github.com/user-attachments/assets/6d877c76-e060-4105-adbf-505dbe1e1283)
